app/views/home/first_login_help.html.erb

```diff
@@ -1,49 +1,233 @@
-<p>To add your info, go to <%= link_to "your home page", user_path(current_user) %>, select the "<%= link_to "Profile", user_profile_path(current_user) %>" tab, and select "<%= link_to "Edit My Profile", edit_user_path(current_user) %>" from the range of options at the end of the page. Here, you can enter some basic personal information. It's also where to go to change your password.</p>
-
-<p>"<%= link_to 'Set My Preferences', user_preferences_path(current_user) %>" is located next to "Edit My Profile", and is pretty self-explanatory. The Preferences area controls both the Archive's behavior (such as whether your viewing history is saved) and the Archive's appearance. The Archive's appearance controls will get more and more sophisticated as we get closer and closer to <acronym title="Archive of Our Own">AO3</acronym>'s first full release; i.e. if you really hate the font(s) we use, you will be able to change them soon. You can also access your preferences from your Dashboard.</p>
-
-<h3 id="support">Support and Feedback</h3>
-<p>As you use the archive, never forget that we're still in beta and that we love, love, love feedback. Please send us some, at our <%= link_to "handy-dandy feedback page", new_feedback_report_path %>, as often as you like.</p>
-<p>If there's anything more you want to know, then please visit our <%= link_to "FAQ", archive_faqs_path %> or contact our <%= link_to "support team", new_feedback_report_path %>.</p>
+  <h2 class="heading">
+    <%= ts('Welcome to the %{app_name}!', app_name: ArchiveConfig.APP_NAME) %>
+  </h2>
+  <p><%= ts('Here are some tips to help you get started.') %></p>
+
+  <h3 class="landmark heading"><%= ts('Table of Contents') %></h3>
+  <ul id="toc" class="toc" role="directory">
+    <li><%= link_to ts('Logging In and Logging Out'), '#logging_in' %></li>
+    <li>
+      <%= link_to ts('Editing Your Profile, Password, and Preferences'),
+                  '#edit_profile' %>
+    </li>
+    <li><%= link_to ts('Pseuds'), '#pseuds' %></li>
+    <li><%= link_to ts('Posting Works'), '#posting' %></li>
+    <li><%= link_to ts('Browsing'), '#browsing' %></li>
+    <li><%= link_to ts('Tags'), '#tags' %></li>
+    <li><%= link_to ts('Warnings'), '#warnings' %></li>
+    <li><%= link_to ts('Bookmarking Works'), '#bookmarking' %></li>
+    <li><%= link_to ts('Preferences'), '#preferences' %></li>
+    <li>
+      <%= link_to ts('Additional Browsing Information'), '#additional' %>
+      <ul>
+        <li><%= link_to ts('History and Mark for Later'), '#history' %></li>
+        <li><%= link_to ts('Subscriptions'), '#subscriptions' %></li>
+      </ul>
+    </li>
+    <li><%= link_to ts('Terms of Service'), '#legal_stuff' %></li>
+    <li><%= link_to ts('Support and Feedback'), '#support' %></li>
+  </ul>
+
+  <h3 class="heading" id="logging_in">
+    <%= ts('Logging In and Logging Out') %>
+  </h3>
+  <p><%= ts('To log in, locate the login link and fill in your Username and 
```

Comment:
As this is text approved by several people, I assume this is how we want it, but the capitalisation looks a bit weird. (Maybe "fill in the Username and Password fields" would be better for next time)